src/cli.js

```diff
@@ -46,7 +46,8 @@ if (require.main === module) {
         .option('--inject <value>', 'path to a file to be injected', collect, [])
         .option('--full-screen', 'if the app should always be started in full screen')
         .option('--maximize', 'if the app should always be started maximized')
-        .option('--hide-window-frame', 'disable window frame and controls')
+        .option('--hide-window-frame', 'disable window frame and controls, should not be specified with --hide-title-bar together')
```

Comment:
This option didn't added by me. It's from another pull request, and is used at https://github.com/tete1030/nativefier/blob/9bf593761a4727b43bf1f437fd4d5938e735158d/src/options/optionsMain.js#L63
The only thing I did was adding `--hide-title-bar` feature. But this feature conflicts with `--hide-window-frame` so I changed the description to notify user.
About the added line at Line 35 of `docs/api.md`, I think this line was missed by the person who added the `--hide-window-frame` feature, I just added missing part.